rinex/db/NAV/orbits.json

```diff
@@ -680,20 +688,22 @@
             "idot": "f64",
             "deltaN0Dot": "f64",
             "satType": "f64",
+            "t_op": "f64",
             "sisa1_oe": "f64",
             "sisa1_ocb": "f64",
             "sisa1_oc1": "f64",
             "sisa1_oc2": "f64",
-            "isc_b1cd": "f64",
-            "tgd_b1cp": "f64",
-            "tgd_b2ap": "f64",
+            "iscB1Cd": "f64",
+            "spare1": "xxx",
+            "tgdB1Cp": "f64",
+            "tgdB2ap": "f64",
             "sismai": "f64",
             "health": "f64",
-            "b1c": "f64",
+            "integrityB1C": "f64",
```

Comment:
I ended up putting `integrity` first, to match `isc` and `tgd` - means I can use the camel case in the signal name

If you prefer to keep the naming from the RINEX specification then I can put integrity last but then would have to make lower case `b`

Reply:
no that's alright, IMO it's better to have consistent names at the expense of grammar correctness